uiowa/uiowa
# Feature: Immunology updates and additional person_extended tweaks

Resolves #2004 

Additionally, some updated config for iowasuperfund.uiowa.edu.

Finally, a minor update to the LB UI to adjust the look of blocks that are dragged and dropped.

# To test
* `composer install`
* `blt ds --site=immuno.grad.uiowa.edu`
* Notice there are no warnings during sync.
